mhkit/tidal/__init__.py

```diff
@@ -1,3 +1,9 @@
+"""
+MHKiT Tidal Module
+
+This module provides tools for analyzing and processing tidal data.
```

Comment:
Copying slightly longer sentence from our API docs:
```suggestion
The tidal module contains a set of functions to calculate quantities of interest for tidal energy converters (TEC). The tidal module uses timeseries data of velocity and direction.